lib/node_modules/@stdlib/stats/base/dists/pareto-type1/pdf/benchmark/benchmark.js

```diff
@@ -59,18 +68,24 @@ bench( pkg+':factory', function benchmark( b ) {
 	var mypdf;
 	var alpha;
 	var beta;
+	var len;
 	var x;
 	var y;
 	var i;
 
 	alpha = 100.56789;
 	beta = 55.54321;
 	mypdf = pdf.factory( alpha, beta );
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( EPS, 50.0 );
+	}
+	mypdf = pdf.factory( alpha, beta );
 
```

Comment:
```suggestion
	mypdf = pdf.factory( alpha, beta );
	len = 100;
	x = new Float64Array( len );
	for ( i = 0; i < len; i++ ) {
		x[ i ] = uniform( EPS, 50.0 );
	}
```

The same line is repeated again, which is not needed.